lib/dat-json.js

```diff
@@ -0,0 +1,42 @@
+var fs = require('fs')
+var stringKey = require('dat-encoding').toStr
+var path = require('path')
+
+module.exports = function (dat) {
+  return {
+    read: function (cb) {
+      // dat.json
+      // reads to dat.meta if exists
+      // (TODO: move to module & validate dat.json)
+      fs.readFile(datJsonFile(dat), 'utf8', function (err, body) {
```

Comment:
Should this go to disk, or rather read meta from an archive live stream? This way we could also implement a follow mode where you subscribe to meta updates.